test/rsvp.js

```diff
@@ -0,0 +1,233 @@
+            testUser = User.forge({ id: 1, email: 'new@example.com' });
+            testUser.related('roles').add({ role: utils.roles.ATTENDEE });
+
+            testAttendee = Attendee.forge({
+                "id": 1,
+                "userId": 1,
+                "firstName": "John",
+                "lastName": "Doe",
+                "shirtSize": "M",
+                "diet": "NONE",
+                "age": 19,
+                "graduationYear": 2019,
+                "transportation": "NOT_NEEDED",
+                "school": "University of Illinois at Urbana-Champaign",
+                "major": "Computer Science",
+                "gender": "MALE",
+                "professionalInterest": "BOTH",
+                "github": "JDoe1234",
+                "linkedin": "JDoe5678",
+                "interests": "CS",
+                "isNovice": true,
+                "isPrivate": false,
+                "hasLightningInterest": false,
+                "phoneNumber": "12345678910",
+                "ecosystemInterests": [
+                    {
+                        "ecosystemId": 1,
+                        "attendeeId": 1,
+                        "id": 1
+                    }
+                ],
+                "collaborators": [
+                    {
+                        "collaborator": "collaborator@hackillinois.org",
+                        "attendeeId": 1,
+                        "id": 1
+                    }
+                ],
+                "extras": [
+                    {
+                        "info": "One of the projects I'm really proud of is my HelloWorld Machine. It says 'Hello World' in ten different languages!",
+                        "attendeeId": 1,
+                        "id": 1
+                    }
+                ]
+            });
+
+            testRSVP =  {
+                "id": 1,
+                "attendeeId": 1,
+                "isAttending": true,
+                "type": "CREATE"
+            };
+
+            _forgeRSVP = sinon.spy(AttendeeRSVP, 'forge');
+            _saveRSVP = sinon.spy(AttendeeRSVP.prototype, 'save');
+
+            done();
+        });
+        beforeEach(function (done) {
+            tracker.install();
+            done();
+        });
+        it('creates a rsvp for a valid attendee and sets its attendee role',function(done){
+            var testRSVPClone = JSON.parse(JSON.stringify(testRSVP));
+
+            tracker.on('query', function (query) {
+                query.response([]);
+            });
+
+            var RSVP = RSVPService.createRSVP(testAttendee, testUser, testRSVPClone);
+            RSVP.bind(this).then(function() {
+                var _userRole = testUser.getRole(utils.roles.ATTENDEE);
+
+                assert(_forgeRSVP.called, "RSVP forge not called with right parameters");
+                assert(_saveRSVP.calledOnce, "RSVP save not called");
+                assert(_userRole.get('active'), "AttendeeRole not set to active");
```

Comment:
`AttendeeRole` isn't really a thing. Can we just change this to `Attendee role is not set to active`?